aria/workflow_engine/tasks.py

```diff
@@ -113,11 +113,11 @@ def requires(self):
         """
         The operations and sub-workflows that this task depends on.
         """
-        for i, task in enumerate(self.depends_on):
+        for task in self.depends_on:
             yield task
 
 
-class StartWorkflowTask(EngineBaseTask):
+class StartWorkflowTask(EngineBaseTask, WrapperTask):
```

Comment:
use EngineWrapperTask instead on EngineBaseTask, WrapperTask
+ do we still need EngineBaseTask and EngineWrapperTask or can we combine them?